python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -947,9 +947,19 @@ def plotRhoStats(axes, rhoStats):
         colorStr = "C{}".format(rhoIndex)
         xi = rho.xip if rhoIndex > 0 else rho.xi
         isPositive = xi > 0
-        ax.errorbar(rho.meanr[isPositive], xi[isPositive], yerr=np.sqrt(rho.varxi)[isPositive],
+        #  KK/GGCorrelation objects have different entries for variance in x
+        if hasattr(rho, "varxi"):
+            yErrPositive = np.sqrt(rho.varxi)[isPositive]
+            yErrNotPositive = np.sqrt(rho.varxi)[~isPositive]
+        elif hasattr(rho, "varxip"):
+            yErrPositive = np.sqrt(rho.varxip)[isPositive]
+            yErrNotPositive = np.sqrt(rho.varxim)[~isPositive]
```

Comment:
This should also be `rho.varxip` since we compute only `xip` and its variance. The `isPositive` is merely a split based on whether the `xip` is positive or not, so that we can mark them appropriately when the axis is in log-scale.